app/Console/Commands/FetchCollectionNfts.php

```diff
@@ -33,25 +32,26 @@ class FetchCollectionNfts extends Command
     public function handle(): int
     {
         $onlySigned = (bool) $this->option('only-signed');
+        $limit = $this->option('limit');
 
         if ($onlySigned) {
             Collection::getWithSignedWallet()->each(function (Collection $collection) {
-                if (! $collection->isBlacklisted()) {
-                    FetchCollectionNftsJob::dispatch(
-                        $collection,
-                        $this->option('start-token') ?? $collection->last_indexed_token_number
-                    );
-                }
+                FetchCollectionNftsJob::dispatch(
+                    $collection,
+                    $this->option('start-token') ?? $collection->last_indexed_token_number
+                );
             });
         } else {
-            $this->forEachCollection(function ($collection) {
-                if (! $collection->isBlacklisted()) {
+            $this->forEachCollection(
+                callback: function ($collection) {
                     FetchCollectionNftsJob::dispatch(
                         $collection,
                         $this->option('start-token') ?? $collection->last_indexed_token_number
                     );
-                }
-            }, queryCallback: fn (Builder $query) => $query->withAcceptableSupply());
+                },
+                queryCallback: fn ($query) => $query->orderByOldestNftLastFetchedAt(),
```

Comment:
You need to keep the `supply` checks in place, otherwise it will try to pull NFTs collections with 20K supplu which will drain our API budget.